src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### Changed Rules
+Rule ID | New Category | New Severity | Old Category | Old Severity | Notes
+--------|--------------|--------------|--------------|--------------|-------
+CA1416 | Interoperability | Warning | Interoperability | Info | PlatformCompatabilityAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1416)
```

Comment:
You can revert change to this file and instead manually update the entry in shipped file. We would normally not do that but this is a special case as this analyzer is shipping in 5.0 release.

Reply:
Ok